ui/admin/app/components/form/credential-library/vault-generic/index.hbs

```diff
@@ -108,6 +108,38 @@
     {{/if}}
   </Hds::Form::TextInput::Field>
 
+  <Hds::Form::Select::Field
+    @isOptional={{true}}
+    @width='100%'
+    name='credential_type'
+    disabled={{or form.disabled (not @model.isNew)}}
+    {{on 'input' (set-from-event @model 'credential_type')}}
+    as |F|
+  >
+    <F.Label>
+      {{t 'resources.credential-library.form.credential_type.label'}}
+    </F.Label>
+    <F.HelperText>
+      {{t 'resources.credential-library.form.credential_type.help'}}
+    </F.HelperText>
+    <F.Options>
+      <option selected value='' />
```

Comment:
We can't always have this set as `selected` right? Don't we still need to have it check if it should be selected if it's now a real selection?

Reply:
theoretically yes? But it looks like the selection seems to work fine when I go back and forth with the options 🤔

Reply:
You're technically right and it looks like it still works (I checked and the DOM _property_ for `selected` is correctly set to false, which is what matters, even though the DOM _attribute_ for selected is still there) but I'm worried about future bugs in the future. It might be good enough to just not include the attribute at all?

Reply:
I removed the attr to avoid any bugs, tested and works as expected with no attr.